config.json

```
@@ -2380,6 +2380,21 @@
         ],
         "difficulty": 9
       },
+      {
+        "slug": "satellite",
+        "name": "Satellite",
+        "uuid": "a07e01e6-7cea-4db9-a514-7c95788e90a9",
+        "prerequisites": [
+          "pattern-matching",
+          "tuple",
```

The correct concept name is:
```suggestion
          "tuples",
```